resources/views/hr/payroll/index.blade.php

```diff
@@ -82,64 +82,139 @@
+                        <td class="table-item-content">{{$payroll->total_overtime}} Birr</td>
+                        <td class="table-item-content">{{$payroll->total_loan}} Birr</td>
+                        <td class="table-item-content">{{$payroll->total_tax}} Birr</td>
+                        <td class="table-item-content">{{$payroll->total_pension_7}} Birr</td>
+                        <td class="table-item-content">{{$payroll->total_pension_11}} Birr</td>
+                        <td class="table-item-content">{{$payroll->net_pay}} Birr</td>
+
                     </tr>
+                    @endforeach
                 </tbody>
             </table>
         </div>
     </div>
 </div>
 
+{{-- Select Accounting Period Modal --}}
+<div class="modal fade" id="modal-select-period" tabindex="-1" role="dialog" aria-labelledby="modal-select-period" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="modal-select-period">Select Accounting Period</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <form action="{{route('payrolls.payrolls.store')}}" method="post">
+                    @csrf
+                    {{-- <div class="form-group row">
+                        <label for="employee" class="col-12 col-md-3">Employee</label>
+                        <div class="col-12 col-lg-6">
+                            <select class="form-control select2" id="employee" name="employee">
+                                <option>Graeme Xyber Pastoril</option>
+                                <option>Justin Manigo</option>
+                                <option>Lester Fong</option>
+                            </select>
+                        </div>
+                    </div> --}}
+                    <div class="form-group row  mb-3">
+                        <label for="month" class="col-12 col-md-4">Accounting Period</label>
+                        <div class="col-12 col-lg-8">
+                            <select class="form-control" id="period" name="period">
+                                @if($payrolls_this_year->count() > 0)){
+                                    {{-- display last record of $periods + 1--}}
+                                    <option value="{{$payrolls_this_year ->last()->period_id+1}}">Period {{$payrolls_this_year ->last()->period_id+1}}</option>
+                                    }
+                                @else{
+                                    <option value="1">Period 1</option>
+                                }
+                                @endif
```

Comment:
I think you can create another table to store which accounting periods have payrolls generated. See `accounting_periods` table for your reference ;)